src/Microsoft.Dnx.Tooling/Utils/PackageFeedFactory.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Concurrent;
+using Microsoft.Dnx.Tooling.Restore.NuGet;
+using NuGet;
+
+namespace Microsoft.Dnx.Tooling
+{
+    public static class PackageFeedFactory
```

Comment:
This should be non static and created once and passed around in the RestoreCommand